pkg/webhooks/resource/handlers.go

```diff
@@ -43,7 +43,8 @@ type handlers struct {
 	metricsConfig *metrics.MetricsConfig
 
 	// cache
-	pCache policycache.Cache
+	pCache                 policycache.Cache
+	informerCacheResolvers enginectx.ConfigmapResolver
```

Comment:
Move this in the policy context builder.

Reply:
fixed